### PR TITLE
Update forced-colors Chrome version status

### PR DESCRIPTION
**Pull request by dlibby-**
Forced colors mode is shipping in Chrome 89. Updating the version information to reflect this.

See: 
https://chromium.googlesource.com/chromium/src/+/0b1c596b6732f6bc9d440b12af53d3c74aa3c4ac
https://www.chromestatus.com/feature/5757293075365888#details
https://blog.chromium.org/2021/01/chrome-89-beta-advanced-hardware.html

I'm also working on a PR to update the MDN docs for these CSS features.

### CODE REVIEW

**Comment by dlibby-** [comment]
@foolip - PTAL thanks! (I don't seem to have permissions to add a reviewer myself)

File: css/at-rules/media.json [hunk 1]
```diff
@@ -612,15 +612,7 @@
...
-                "version_added": "79",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "#forced-colors",
-                    "value_to_set": "Enabled"
-                  }
-                ],
-                "notes": "See <a href='https://crbug.com/970285'>bug 970285</a>."
+                "version_added": "89"
```
Comment (foolip) [inline css/at-rules/media.json:615]: I am very happy to see flag data removed, but the [guideline](https://github.com/mdn/browser-compat-data/blob/master/docs/data-guidelines.md#removal-of-irrelevant-flag-data) is to do so only after the feature has been enabled by default for two years. So, what you'll need to do here instead is to use an array of two support statements, also adding `"version_added": "89"` for the flags entry.

Comment (dlibby-) [reply css/at-rules/media.json]: Apologies for missing the guidance (should have read more closely). This is fixed now.

Comment (foolip) [reply css/at-rules/media.json]: I doubt anyone reads the guidelines before a specific one is relevant in a PR, I haven't read all of them at least :)

File: css/at-rules/media.json [hunk 2]
```diff
@@ -611,17 +611,22 @@
...
+                    {
+                      "type": "preference",
+                      "name": "#forced-colors",
+                      "value_to_set": "Enabled"
+                    }
+                  ],
+                  "notes": "See <a href='https://crbug.com/970285'>bug 970285</a>."
+                },
+                {
+                  "version_added": "89"
```
Comment (foolip) [inline css/at-rules/media.json:627]: Unfortunately the order of entries matter, and the first one is the one that'll be shown on MDN by default. So you'll need to change the order to put this first.

Comment (dlibby-) [reply css/at-rules/media.json]: Oops! Fixed. Thanks for bearing with me!

**Review by foolip — APPROVED** [review]

**Comment by foolip** [comment]
Thanks @dlibby-!

**Pull request merged**